tests/builder/builder_test.py

```diff
@@ -1009,6 +1010,53 @@ def test_glyph_lib_Export_GDEF(self):
         ufo2 = ds2.sources[0].font
         self.assertEqual(ufo2.features.text, "")
 
+    def test_glyph_lib_Export_feature_names_from_notes(self):
+        font = generate_minimal_font()
+        add_glyph(font, "a")
+        add_glyph(font, "a.ss01")
+        ss01 = GSFeature(name="ss01", code="sub a by a.ss01;")
+        font.features.append(ss01)
+        for note in (
+            'Name: Single\\storey "ä"',
+            'A Comment\nName: Single\\storey "ä"\nFoo',
+            'A Comment\nName: Single\\storey "ä"',
+            'Name: Single\\storey "ä"\nFoo',
+        ):
+            font.features[0].notes = note
+            ufos = self.to_ufos(font)
+            ufo = ufos[0]
+            self.assertIn(r'name "Single\005cstorey \0022ä\0022";', ufo.features.text)
+
+    def test_glyph_lib_Export_feature_names_long_from_notes(self):
+        font = generate_minimal_font()
+        add_glyph(font, "a")
+        add_glyph(font, "a.ss01")
+        ss01 = GSFeature(name="ss01", code="sub a by a.ss01;")
+        font.features.append(ss01)
+        for note in (
+            'featureNames {\n  name 3 1 0x401 "Alternate {ä};";\n  name 3 1 0x409 "Alternate {};";\n};\n',
```

Comment:
it would be good if you could wrap these very long strings here and further below into multiple lines